pallets/gear/src/weights.rs

```diff
@@ -959,6 +990,12 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
             // Standard Error: 63_779
             .saturating_add(Weight::from_ref_time(9_522_432).saturating_mul(r.into()))
     }
+    /// The range of component `r` is `[0, 50]`.
+    fn instr_i32load(r: u32, ) -> Weight {
```

Comment:
could you copy weights from x64 analogs for newly added weights or do proper benches on special hardware we use.
these seem to be benched on your local machine, isn't it?